docs/dialect.md

```diff
@@ -94,6 +95,33 @@ Output:
 SELECT * FROM `test` WHERE `id` = 10 []
 ```
 
+<a name="sqlserver"></a>
+### SQLServer
+```go
+import (
+  "fmt"
+  "github.com/doug-martin/goqu/v9"
+  // import the dialect
+  _ "github.com/doug-martin/goqu/v9/dialect/sqlserver"
+)
+
+// look up the dialect
+dialect := goqu.Dialect("sqlserver")
+
+// use dialect.From to get a dataset to build your SQL
+ds := dialect.From("test").Where(goqu.Ex{"id": 10})
+sql, args, err := ds.ToSQL()
+if err != nil{
+  fmt.Println("An error occurred while generating the SQL", err.Error())
+}else{
+  fmt.Println(sql, args)
+}
+```
+
+Output:
+```
+SELECT * FROM `test` WHERE `id` = 10 []
```

Comment:
Is this the correct output, I'm not familiar with SQL server dialect but from your tests it looks like `"` should be used instead of backtick

Reply:
My bad, just copied from previous example, and looked so trivial that I forgot about `"`
Will fix